config/nav.yml

```diff
@@ -220,12 +219,12 @@ nav:
           - Overview: eventing/samples/parallel/README.md
           - Multiple Cases: eventing/samples/parallel/multiple-branches/README.md
           - Mutual Exclusivity: eventing/samples/parallel/mutual-exclusivity/README.md
-        - CloudAuditLogsSource: eventing/sources/cloud-audit-logs-source/README.md
-        - CloudPubSubSource: eventing/sources/cloud-pubsub-source/README.md
-        - CloudSchedulerSource: eventing/sources/cloud-scheduler-source/README.md
-        - CloudStorageSource: eventing/sources/cloud-storage-source/README.md
-        - GitHub source: eventing/sources/github-source/README.md
-        - GitLab source: eventing/sources/gitlab-source/README.md
+        - CloudAuditLogsSource: eventing/samples/cloud-audit-logs-source/README.md
+        - CloudPubSubSource: eventing/samples/cloud-pubsub-source/README.md
+        - CloudSchedulerSource: eventing/samples/cloud-scheduler-source/README.md
+        - CloudStorageSource: eventing/samples/cloud-storage-source/README.md
+        - GitHub source: eventing/samples/github-source/README.md
+        - GitLab source: eventing/samples/gitlab-source/README.md
```

Comment:
Same for all of these, idk why they wouldn't be part of the single Event Source docs section?

Reply:
I was confused about what to do with samples. There were three options I could think of (I was not sure which one we plan to do so I just left them where they were):

- Do we plan to have a section of the docs for all samples? 
- Or a subsection for all samples at the end of each guide? 
- Or do we want to have samples at the end of each subgroup -- eg, event source samples, autoscaling samples? 

I think if we have all the samples in the same place it might be easier to link to from the What's Next page, but this is not a strongly held position.

Reply:
I would prefer we don't have a samples section at all and just add them as part of the main docs for each thing they're a sample of. As a user I wouldn't want to look in two different places for docs on how to create a resource and then samples for them. For example, rather than having a GitLab source sample somewhere, just add GitLab source to the main event source docs and use the sample as a basis for documenting it properly.